ultravox/inference/base.py

```diff
@@ -10,6 +12,8 @@ class VoiceOutput:
     text: str
     input_tokens: int
     output_tokens: int
+    audio_token_len: int = 0
```

Comment:
maybe call this `input_audio_tokens` for consistency

Reply:
This field is no longer needed after adding history support inside LocalInference